public/js/admin.js

```diff
@@ -244,13 +252,13 @@ var submitUserForm = function() {
         data: user,
         success: function(data) {
             if (data == 'failure') {
-                $('#result').html('User could not be added');
+		dropSnack(colours.FAIL_RED, 'User could not be added');
```

Comment:
Maybe make this dropSuccessSnack(message)
dropErrorSnack(message) etc.
Right now the way you know if the snack represent a success or failure is by its color.

Reply:
We'll they'd just be repeated blocks of code, the only thing that differs is the colour and the text, both of which are parameters of the dropSnack function. If we ever want to add more colours to represent something else we can just add them to the list at the top and call it a day.

Unless I'm missing some logic here X'(